package.json

```diff
@@ -22,8 +22,7 @@
     "bole": "2.0.0",
     "dotenv": "2.0.0",
     "hapi": "13.3.0",
-    "hoek": "3.0.4",
-    "moment-tokens": "1.0.0"
```

Comment:
Since we're no longer treating the message as the canonical source, we don't need to format the time to match Common Log Format. We already have the time in all of our logs via bole.